runtime/doc/nvim_terminal_emulator.txt

```diff
@@ -117,8 +117,8 @@ variables:
 - *b:term_title* The settable title of the terminal, typically displayed in
   the window title or tab title of a graphical terminal emulator. Programs
   running in the terminal can set this title via an escape sequence.
-- *b:terminal_job_id* The nvim job ID of the job running in the terminal. See
-  |job-control| for more information.
+- *b:terminal_channel_id* The nvim channel ID for the underlying PTY.
+  |chansend()| can be used to send input to the terminal.
```

Comment:
If we're going to introduce a new variable it should be `b:term_` prefix like `b:term_title`.

But I'm not sure these need to be terminal-specific: why not have buffer-local `'jobid'` or `'chanid'` options which control what job drives a buffer?